markdown/3.x/en/core-result-dispatcher.md

```diff
@@ -107,3 +107,18 @@ registered callback.
 </script>
 <script defer src="pathToFFWebComponents/dist/bundle.js"></script>
 ```
+
+### `dispatchRaw(response, topics)`
+___
+Manually dispatches a response to the Web Components.
+Topics argument is optional - skipping it causes the response will be dispatched as standard search result.
```

Comment:
Perhaps we could elaborate a bit on this.
For example why this method is useful, what its intended use case is and so on.
How does its behaviour differ when specifying `topics` compared to not specifying it?
If the response is not dispatched as a standard search result, what is it dispatched as instead?

Reply:
I updated the description. Hope that this level of details makes things more clear with not confusing reader in the same time